.github/workflows/github-commit.yml

```
@@ -0,0 +1,74 @@
+
+on:
+  # Ref: <https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#release>.
+  push:
+    branches: ["release"]
+
+jobs:
+  prelude:
+    runs-on: ubuntu-latest
+
+    outputs:
+      tag: ${{ steps.get_version.outputs.tag }}
```

actionlint (1.7.4)

property "get_version" is not defined in object type {}

(expression)